### DISCUSSION TITLE
React v18 Peer Depedency

### DISCUSSION THREAD

**Discussion by lsbyerley** [resend/react-email#1543] [Q&A]
Would love to use the `@react-email/components` library in a project that is still on react v17. Is there a prior version of react-email that does not require react v18?

**Comment by gabrielmfern** [comment]
Not that I can remember, but I do believe you can use [overrides](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#overrides) or [yarn resolutions](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) to force `@react-email/components` into using React 17. I do think it would work properly.

**Reply by gabrielmfern** [reply]
Yeah, no version we ever had supported React 17. To support it properly, `@react-email/render` would probably be the blocker here.
